apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -142,17 +143,33 @@ model GuildContributionVerificationStatus {
 }
 
 model GuildActivityType {
-  id               Int          @id @default(autoincrement())
-  createdAt        DateTime     @default(now())
-  updatedAt        DateTime     @default(now()) @updatedAt
-  guild_id         Int
-  guild            Guild        @relation(fields: [guild_id], references: [id])
-  activity_type_id Int
-  activity_type    ActivityType @relation(fields: [activity_type_id], references: [id])
+  id                         Int                          @id @default(autoincrement())
+  createdAt                  DateTime                     @default(now())
+  updatedAt                  DateTime                     @default(now()) @updatedAt
+  guild_id                   Int
+  guild                      Guild                        @relation(fields: [guild_id], references: [id])
+  activity_type_id           Int
+  activity_type              ActivityType                 @relation(name: "guilds", fields: [activity_type_id], references: [id])
+  migrated_from_id           Int?
+  migrated_from              ActivityType?                @relation(name: "migrated_from", fields: [migrated_from_id], references: [id])
```

Comment:
In this case, we need to define custom names, since we have two relations referencing the same table. both are from `GuildActivityType` -> `ActivityType`, and the prisma linter considers it an error to not name the relations manually. although we could probably just name the second/new relation as opposed to both.

Reply:
Ahaa. Thank you for bringing this up.. I missed this point.